src/translations/en.json

```diff
@@ -1760,6 +1760,7 @@
             "password": "Password",
             "password_confirm": "Confirm Password",
             "password_not_match": "Passwords don't match",
+            "password_changed": "Password changed successfully",
```

Comment:
The text already exists, see line 1740 so no need to add a new key.
And please change the existing key to say "was changed" since that is more logical and also the way the existing translations worded it (this key is not used anywhere yet).

Reply:
I will direct it to use the line on 1740 and update it accordingly. Also, I'm trying to figure out how to remove those commits. Sorry about that.

Reply:
I'm sorry. I don't know how to remove those commits. I'm just starting out, so I haven't encountered this issue before. Not sure what happened.